src/rules/prefer-lodash-method.js

```diff
@@ -88,7 +105,11 @@ module.exports = {
             }),
             'CallExpression:exit'(node) {
                 if (!usingLodash.has(node) && !shouldIgnore(node) && canUseLodash(node)) {
-                    context.report(node, `Prefer '_.${getMethodName(node)}' over the native function.`)
+                    let lodashMethodName = getMethodName(node)
+                    if (isNativeStringMethodCall(node)) {
+                      lodashMethodName = get(nativeStringMap, lodashMethodName)
```

Comment:
No need for `get` here. At this point you can just do a regular property access, since the `if` guards for it.